### PR TITLE
Add swagger docs specification for internal/operator APIs

### PR DESCRIPTION
**Pull request by mingfeng**
Some small details need to be added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,15 @@ Operator API will be available at [http://127.0.0.1:8000/operator/v1/](http://12
 
 Internal API will be available at [http://127.0.0.1:8000/internal/v1/](http://127.0.0.1:8000/internal/v1/)
 
+### Generating API documentation
+
+The API documentation conforms to [Swagger Specification 2.0](http://swagger.io/specification/). Use
+[swagger-codegen](https://github.com/swagger-api/swagger-codegen) to generate the documentation.
+
+For example, to generate static HTML documentation for operator APIs:
```
Comment (jaywink) [inline README.md]: I would add some more info, for example something as follows. I love verbose instructions ;)

    You need Maven to build the Swagged-Codegen client:

        # Ubuntu
        sudo apt-get install maven

    Due to [a bug in Codegen](https://github.com/swagger-api/swagger-codegen/pull/4508), we're using an unreleased version at the moment. Clone Codegen master branch and build it.

        git clone https://github.com/swagger-api/swagger-codegen 
        cd swagger-codegen/
        mvn clean package  # Takes a few minutes

    The client will now be available at `modules/swagger-codegen-cli/target/swagger-codegen-cli.jar`.

    To build the docs, in `parkkihubi` repository root:

        cd docs/api
        java -jar /path/to/codegen/swagger-codegen-cli.jar generate -i internal.yaml -l html2 -c config.json -o /output/path/internal/
        java -jar /path/to/codegen/swagger-codegen-cli.jar generate -i operator.yaml -l html2 -c config.json -o /output/path/operator/

Comment (mingfeng) [inline README.md]: This is very nice feedback, thanks!

File: docs/api/internal.yaml [hunk 2]
```diff
@@ -0,0 +1,179 @@
+swagger: '2.0'
+info:
+  title: Parkkihubi
+  description: Parkkihubi REST API Documentation (Internal)
+  version: "1.0.0"
```
Comment (tuomas777) [inline docs/api/internal.yaml]: I would rather use "0.0.1" ATM :D

File: docs/api/operator.yaml [hunk 3]
```diff
@@ -0,0 +1,254 @@
+swagger: '2.0'
+info:
+  title: Parkkihubi
+  description: Parkkihubi REST API Documentation (Operator)
+  version: "1.0.0"
```
Comment (tuomas777) [inline docs/api/operator.yaml]: ditto.

File: docs/api/operator.yaml [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+      operationId: createParking
+      parameters:
+        - name: parking
+          in: body
+          required: true
+          description: Parking to add to the system
+          schema:
+            $ref: '#/definitions/Parking'
+      responses:
+        200:
```
Comment (tuomas777) [inline docs/api/operator.yaml]: This should be 201.

File: docs/api/operator.yaml [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+        - ApiKey: []
+      tags:
+        - parking
+      summary: Update a parking by its ID
+      operationId: updateParkingById
+      parameters:
+        - name: parking_id
+          in: path
+          required: true
+          description: ID of the parking to udpate
```
Comment (tuomas777) [inline docs/api/operator.yaml]: *update

File: docs/api/operator.yaml [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+          format: uuid
+        - name: parking
+          in: body
+          required: true
+          description: Parking to add to the system
+          schema:
+            $ref: '#/definitions/Parking'
+      responses:
+        200:
+          description: The udpated parking
```
Comment (tuomas777) [inline docs/api/operator.yaml]: *updated

File: docs/api/operator.yaml [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+        - ApiKey: []
+      tags:
+        - parking
+      summary: Partial update a parking by its ID
+      operationId: partialUpdateParkingById
+      parameters:
+        - name: parking_id
+          in: path
+          required: true
+          description: ID of the parking to udpate
```
Comment (tuomas777) [inline docs/api/operator.yaml]: *update

File: docs/api/operator.yaml [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+          format: uuid
+        - name: parking
+          in: body
+          required: true
+          description: Parking to add to the system
+          schema:
+            $ref: '#/definitions/Parking'
+      responses:
+        200:
+          description: The udpated parking
```
Comment (tuomas777) [inline docs/api/operator.yaml]: *updated

**Pull request merged**